libs/game/textDialogs.ts

```diff
@@ -241,7 +241,10 @@ namespace game {
                 }
             }
 
-            screens.push(current)
+            //Only pushes the last part of the message to the screen when current isn't empty 
+            if(current !== ""){
```

Comment:
Two nitpicks: 
1. Put a space before the open parentheses and after the close parentheses 
1. The more JavaScript-y way to write this if-statement would be like this:

```typescript

// current will evaluate to false if it is the empty string
if (current) {

}

```